ui/src/components/QMarkdown.js

```diff
@@ -408,7 +408,7 @@ export default defineComponent({
 
         // get the markdown - slot overrides 'src'
         let markdown = source.value
-        if (slots.default()[ 0 ].children.trim().length > 0) {
+        if (slots.defaults !== undefined && slots.default()[ 0 ].children.trim().length > 0) {
```

Comment:
Looking good, but this should not be `slots.defaults` - just remove the `s` so I can merge. Thanks.

Reply:
Good catch! I removed the s so you should be good to go.